packer/scripts/firstuser.sh

```diff
@@ -28,7 +28,8 @@ startServices() {
     pids=`pidof node`
     if [ `expr length "$pids"` -eq "0" ]; then
        echo "starting rackhd ./src services..."
-       sudo nf start > /tmp/rackhd.log &
+       sudo pm2 start rackhd.yml 
+       sudo pm2 logs > /tmp/rackhd.log &  
```

Comment:
The log during "pm2 start" and "pm2 logs" will be lost, need to use pm2's log management to collect log: http://pm2.keymetrics.io/docs/usage/log-management/